sklearn/model_selection/_search.py

```diff
@@ -311,11 +311,14 @@ def fit_grid_point(X, y, estimator, parameters, train, test, scorer,
     **fit_params : kwargs
         Additional parameter passed to the fit function of the estimator.
 
-    error_score : 'raise' (default) or numeric
+    error_score : 'raise-deprecating' (default), 'raise' or numeric
```

Comment:
I think of raise-deprecating as an implementation detail that we shouldn't document.

Rather, we need to say that From version 0.22  the default will be np.nan